mlir/include/mlir/Dialect/LLVMIR/LLVMOpBase.td

```diff
@@ -225,6 +225,12 @@ class LLVM_MemOpPatterns {
       inst->setMetadata(llvm::LLVMContext::MD_nontemporal, metadata);
     }
   }];
+  code setInvariantLoadMetedataCode = [{
```

Comment:
Are there plans to add this to more operations? If not, then making this part of the `MemOpPatterns` seems a bit of an overkill.